lib/msal-angular/README.md

```diff
@@ -46,8 +47,8 @@ At a minimum, `@azure/msal-angular` will follow the [support schedule of the mai
 
 | MSAL Angular version | MSAL support status | Supported Angular versions |
 | -------------------- | ------------------- | -------------------------- |
-| MSAL Angular v5      | Active development  | 18, 19, 20                 |
-| MSAL Angular v4      | Active development  | 15, 16, 17, 18, 19, 20     |
+| MSAL Angular v5      | Active development  | 19, 20                     |
```

Comment:
Let's make sure the migration guide mentions which versions we're dropping support for